### PR TITLE
169 change rules page to explain how stuff works

### PR DESCRIPTION
**Pull request by JoMuelle**
Rules Page displays some stuff about the app and some pointers / rules for new users

### CODE REVIEW

File: src/styles/views/Rules.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      font-size: $fontSizeSmall;
+      margin-bottom: 50px;
+     }
+
+    &.smallTitle{
+       font-weight: bold;
+       color: $background;
+       position: relative;
+       font-size: $fontSizeMediumSmall;
+      margin-bottom: -10px;
```
Comment (daveking7) [inline src/styles/views/Rules.scss:30]: -10px is a bit much, would rather change to -5px

**Review by daveking7 — CHANGES_REQUESTED** [review]
Sorry for the many comments. 
1. With browser zoom to 100%, I can scroll to left and to right, IMO there shouldn't be a scrollbar on the x-axis.
2. I think it would be nice to put our names somewhere below
3. for the rest see comments

@serainaschraff Can you look through it too, after Joe has corrected my complaints?

File: src/components/views/Rules.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+
+
+    let information  = (
+        <table className="rules table">
+            <div className="rules largeTitle"> About: </div>
+            <tr className="rules info-container">
+                <td> Date against humanity helps you meet people with the same sense of humour and allows you to network with others in a fun way.
+                    The basic principle is based on the card game "Cards Against Humanity", where you play White Cards to fill the gap in the Black Cards and the funniest player wins.<br/>
+                    You can select your preferences on what players you would like to meet and then find people with the same kind of humor
```
[the remark this answers is not in the document]
Comment (JoMuelle) [reply src/components/views/Rules.js:69]: done

File: src/components/views/Rules.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                </td>
+            </tr>
+            <div className="rules smallTitle"> Black Cards </div>
+            <tr className="rules info-container">
+                <td>Every day you get to choose one Black Card which has a Gap which will be filled by other players by playing White Cards. <br/>
```
[the remark this answers is not in the document]
Comment (JoMuelle) [reply src/components/views/Rules.js:74]: done

File: src/components/views/Rules.js [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+                    Other players can see you Black Card for 24 hours after you choose it.
```
[the remark this answers is not in the document]
Comment (JoMuelle) [reply src/components/views/Rules.js:75]: done

File: src/components/views/Rules.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                    <br/>After 24 hours your {handLink} resets and you get to choose a new Black Card</td>
```
[the remark this answers is not in the document]
Comment (JoMuelle) [reply src/components/views/Rules.js:76]: all done

File: src/components/views/Rules.js [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+
+            </tr>
+            <div className="rules smallTitle"> White Cards </div>
+            <tr className="rules info-container">
+                <td> Each day you get a fixed number of White Cards which you can {playLink} on other Users Black Cards. Try to use them in a way that is as funny or clever as possible.
```
[the remark this answers is not in the document]
Comment (JoMuelle) [reply src/components/views/Rules.js:82]: -

File: src/components/views/Rules.js [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+                    If the other user likes the white card you played you might {matchLink}!
+                    <br/>After 24 hours your {handLink} resets and you get a new set of White Cards</td>
+
+            </tr>
+            <div className="rules smallTitle"> Rating Cards & Matching </div>
+            <tr className="rules info-container">
+
+                <td> Whenever someone has played a White Card on your Black Card you can {rateLink} them by giving either a Thumbs up or Thumbs down. <br/>
```
[the remark this answers is not in the document]
Comment (JoMuelle) [reply src/components/views/Rules.js:90]: them implies card here

File: src/components/views/Rules.js [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+                    When two players give each others White Cards a Thumbs Up they get "matched". Matched Users can chat with each other. <br/>
```
[the remark this answers is not in the document]
Comment (JoMuelle) [reply src/components/views/Rules.js:91]: There is already a link to the chats in the match

File: src/styles/views/Rules.scss [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+     }
+
+  &.largeTitle{
+    font-weight: bold;
+    color: $background;
+    position: relative;
+    font-size: $fontSizeLarge;
+  }
+
+  &.link{
```
[the remark this answers is not in the document]
Comment (JoMuelle) [reply src/styles/views/Rules.scss:40]: done

File: src/styles/views/Rules.scss (hunk 1 above)
Comment (JoMuelle) [reply src/styles/views/Rules.scss:30]: done

File: src/components/routing/routers/GameRouter.js [hunk 10]
```diff
@@ -52,6 +53,11 @@ const GameRouter = props => {
           </GameGuard>
       </Route>
 
+        {/* rules */}
+        <Route exact path={`${props.base}/rules`}>
+            <Rules/>
+        </Route>
```
Comment (daveking7) [inline src/components/routing/routers/GameRouter.js:59]: the Routes were changed to be all in the GameGuard individually, but I'll probably do this directly in dev for Rules

File: src/components/views/Rules.js [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+                <td> Whenever you choose a new Black Card you also get a fixed number of White Cards which you can {playLink} on other Users Black Cards. Try to use them in a way that is as funny or clever as possible.
+                    If the other user likes the white card you played you might {matchLink}!
+                    <br/>After 24 hours your {handLink} resets and you get a new set of White Cards.</td>
+
+            </tr>
+            <div className="rules smallTitle"> Rating Cards & Matching </div>
+            <tr className="rules info-container">
+
+                <td> Whenever someone has played a White Card on your Black Card you can {rateLink} their White Card by giving either a thumbs up or thumbs down. <br/>
+                    When two players give each other's White Cards a thumbs up, they get match. Matched Users can chat with each other. <br/>
```
Comment (daveking7) [inline src/components/views/Rules.js:91]: Can chat also link to MatchOverwiev?

File: src/components/views/Rules.js [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+            <tr className="rules info-container">
+                <td> Whenever you choose a new Black Card you also get a fixed number of White Cards which you can {playLink} on other Users Black Cards. Try to use them in a way that is as funny or clever as possible.
+                    If the other user likes the white card you played you might {matchLink}!
+                    <br/>After 24 hours your {handLink} resets and you get a new set of White Cards.</td>
+
+            </tr>
+            <div className="rules smallTitle"> Rating Cards & Matching </div>
+            <tr className="rules info-container">
+
+                <td> Whenever someone has played a White Card on your Black Card you can {rateLink} their White Card by giving either a thumbs up or thumbs down. <br/>
```
Comment (daveking7) [inline src/components/views/Rules.js:90]: on you Black Card, (comma)

**Review by daveking7 — APPROVED** [review]
What's about the package-lock.json file? Why is it entirely new? 
@serainaschraff Please also have a look at it, as a proficient English speaker ;)

**Pull request merged**